conf/minifi.properties

```diff
@@ -16,6 +16,7 @@
 # Core Properties #
 nifi.version=0.2.0
 nifi.flow.configuration.file=./conf/config.yml
+nifi.accept.dynamic.properties=true
```

Comment:
Any reason why we shouldn't support dynamic properties by default? I understand we are trying to delineate between just blindly accepting new properties, but feel like the model present in NiFi is the best where processors can choose to support it.